admin_manual/configuration_files/external_storage/smb.rst

```diff
@@ -72,6 +72,16 @@ passing the ``-v`` option to the command.
 SMB authentication
 ^^^^^^^^^^^^^^^^^^
 
-In some cases (such as when using login credentials) it's not possible to read the
-smb credentials from the storage configuration, in those cases you can provide
+Update notifications are not supported when using 'Login credentials, save in session' authentication.
+Using login credentials is only supported with 'Login credentials, save in database'.
+
+When using 'Login credentials, save in database' or 'User entered, stored in database' authentication the notify process
+is not able to read the smb credentials from the storage configuration, in those cases you can provide
```

Comment:
This sentence is the opposite of the paragraph before it. Could you check it again?

Reply:
it's correct, although it might look a bit conflicting.

The notify command can't use the '... save in database' credentials because it is not running in the context of a user.
The file scanner can use the credentials because it is run within a user context so it knows which set of credentials to pick

Reply:
Then write it like that and describe what the difference between notify command and file scanner is ;)